pinot-controller/src/main/java/com/linkedin/pinot/controller/helix/SegmentStatusChecker.java

```diff
@@ -221,6 +228,10 @@ public void runSegmentMetrics() {
             nErrors++;
           }
         }
+        if (nOffline < MaxOfflineSegmentsToLog) {
+          LOGGER.warn("Segment {} of table {} has no online replicas", tableName, partitionName);
+          nOffline++;
```

Comment:
Please add nOffline in the log in line 243 as the total number of segments that were found to have lower number of replicas.